server/routes/routeWorkers/standard.js

```diff
@@ -106,15 +106,15 @@ module.exports = ({ formConfig, licenceService, sectionName, nomisPushService, c
     const pushConfig = getIn(formConfig, [formName, 'nomisPush'])
 
     if (getIn(config, ['pushToNomis']) && pushConfig) {
-      await nomisPushService.pushStatus(
-        bookingId,
-        {
-          type: formName,
-          status: !isEmpty(pushConfig.status) ? getIn(updatedLicence, pushConfig.status) : undefined,
-          reason: !isEmpty(pushConfig.reason) ? getIn(updatedLicence, pushConfig.reason) : undefined,
-        },
-        username
-      )
+      const status = !isEmpty(pushConfig.status) ? getIn(updatedLicence, pushConfig.status) : undefined
```

Comment:
It like seems just doing 
`const status = getIn(updatedLicence, pushConfig.status)` 
would be enough as it would return the value or undefined?

Reply:
if pushConfig.status isn't defined then getIn will return the whole licence